### DISCUSSION TITLE
一直显示：提示：网络错误，无法连接到服务器

### DISCUSSION THREAD

**Discussion by lybide** [qiye45/wechatDownload#249] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/7b18e296-1b17-4012-a846-576543e2a04b)

**Comment by lybide** [comment]
![image](https://github.com/user-attachments/assets/3b884e75-f4ca-4aed-b97e-7678c0d19294)

**Reply by qiye45** [reply, marked as answer]
服务器挂了，下载好啦

**Comment by 1eeyung** [comment]
同上面的问题，
![1](https://github.com/user-attachments/assets/eba42154-70b5-42fe-b5cf-114f6ef204bd)
获取ID失败，电脑并未打开梯子
